pkg/commands/workflows/remove_comments.go

```diff
@@ -136,12 +167,21 @@ func (c *RemovePlanCommentsCommand) Process(ctx context.Context) error {
 		}
 
 		for _, comment := range response.Comments {
-			if !strings.HasPrefix(comment.Body, plan.CommentPrefix) {
-				continue
-			}
+			for _, commentType := range c.flagCommentTypes {
```

Comment:
Triple-nested for loop suggest maybe factoring this out into a standalone function

Reply:
Actually,this is fine, never mind